src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/TextAnalyzer.java

```diff
@@ -76,15 +83,17 @@
 	 */
 	public TextAnalyzer(KBController kb){
 		nlpAnalyzer = new StanfordSuite();		
-		//semAnalyzer = new CFGSemanticAnalyzer(kb);
```

Comment:
David, it looks like you added your changes to an older version of the TextAnalyzer (either that or you went through and removed the semantic analyzer). Do a pull and make sure you're working on the correct version. In particular, you can see in this diff that the updated version has a new SPFSemanticAnalyzer()